apps/studio/components/ui/Charts/LogsBarChart.tsx

```diff
@@ -0,0 +1,122 @@
+import dayjs from 'dayjs'
+import { useState } from 'react'
+import { Bar, Cell, BarChart as RechartBarChart, XAxis, YAxis } from 'recharts'
+import { CHART_COLORS, DateTimeFormats } from 'components/ui/Charts/Charts.constants'
+import type { CategoricalChartState } from 'recharts/types/chart/types'
+import type { Datum } from './Charts.types'
+import NoDataPlaceholder from './NoDataPlaceholder'
+import { ChartConfig, ChartContainer, ChartTooltip, ChartTooltipContent, cn } from 'ui'
+
+export const LogsBarChart = ({
+  data,
+  onBarClick,
+}: {
+  data: Datum[]
+  onBarClick?: (datum: Datum, tooltipData?: CategoricalChartState) => void
+}) => {
+  const SIZE = 'tiny'
+  const [focusDataIndex, setFocusDataIndex] = useState<number | null>(null)
+
+  if (data.length === 0) {
+    return (
+      <NoDataPlaceholder
+        message={'No data'}
+        description="It may take up to 24 hours for data to refresh"
+        size={SIZE}
+      />
+    )
+  }
+
+  const startDate = dayjs(data[0]['timestamp']).format(DateTimeFormats.FULL)
+  const endDate = dayjs(data[data?.length - 1]?.['timestamp']).format(DateTimeFormats.FULL)
+
+  return (
+    <div className={cn('flex flex-col gap-y-3')}>
+      <ChartContainer
+        config={
+          {
+            error_count: {
+              label: 'Errors',
+            },
+            ok_count: {
+              label: 'Success',
```

Comment:
Not such a good idea to label it success, as this would include warning logs.
`Other` would work